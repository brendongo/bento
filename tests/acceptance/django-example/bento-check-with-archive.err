Running Bento checks...

✔ 0 findings since last commit in  s
◦ Not showing 1 archived finding(s). To view, run $ cat .bento/archive.json
