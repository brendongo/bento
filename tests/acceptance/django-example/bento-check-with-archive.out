r2c.flake8:
  âœ” No issues detected

